torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13240,6 +13240,13 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
         skips=(
             # lambda impl
             DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),
+            # In-place operations do not play well with forward AD
+            # FIXME it should be fixable, as auto_elementwise works with in-place just fine, and
+            # this one is somewhat similar
+            # INTERNAL ASSERT FAILED Cannot set a va lue of a forward grad if it already exists. 
```

Comment:
Could you file an issue for this?

Reply:
https://github.com/pytorch/pytorch/issues/77447